deployment-scripts/git-set-creds-github.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# bail out if we encounter any errors
+set -e
```

Comment:
Ideally we would probably default to using `set -euo pipefail`. As that would:
- Fail early.
- Fail early even when piping
- Fail when trying to use a variable that hasn't been defined.

Reply:
Good point. Amending